nekoyume/Assets/_Scripts/Game/Battle/RaidStage.cs

```diff
@@ -607,14 +603,16 @@ public void SkipBattle()
 
         private void ClearBattle()
         {
-            if (_battleCoroutine is not null)
+            if (_battleCoroutine != null)
             {
                 StopCoroutine(_battleCoroutine);
                 _battleCoroutine = null;
                 objectPool.ReleaseAll();
             }
             _currentPlayData = null;
-        }
+            _isPlaying = false;
 
+            Time.timeScale = 1;
```

Comment:
```suggestion
            Time.timeScale = 1.25;
```
저희가 실제로 timeScale을 1을 안쓰고 있어서 1.25로 set해줘야합니다

Reply:
d2dfd0b 에서 수정하였습니다